keyboard/src/iosTest/kotlin/KeyboardManagerTests.kt

```diff
@@ -1,30 +1,36 @@
 package com.splendo.kaluga.keyboard
 
+import com.splendo.kaluga.keyboard.IOSKeyboardManagerTests.IOSKeyboardTestContext
+import kotlinx.coroutines.CoroutineScope
 import platform.CoreGraphics.CGRectMake
 import platform.UIKit.UIApplication
 import platform.UIKit.UITextField
 import kotlin.test.assertTrue
 
-class IOSKeyboardManagerTests : KeyboardManagerTests() {
+class IOSKeyboardManagerTests : KeyboardManagerTests<IOSKeyboardTestContext>() {
 
-    private val application = UIApplication.sharedApplication
-    private val textField = MockTextField()
+    inner class IOSKeyboardTestContext(coroutineScope: CoroutineScope) : KeyboardTestContext(), CoroutineScope by coroutineScope {
+        private val application = UIApplication.sharedApplication
+        val textField = MockTextField()
 
-    override val builder get() = KeyboardManager.Builder(application)
+        override val builder get() = KeyboardManager.Builder(application)
 
-    override val view: KeyboardHostingView
-        get() = textField
+        override val view: KeyboardHostingView
+            get() = textField
+    }
 
-    override fun verifyShow() {
+    override fun IOSKeyboardTestContext.verifyShow() {
         assertTrue(textField.didBecomeFirstResponder)
     }
 
-    override fun verifyDismiss() {
+    override fun IOSKeyboardTestContext.verifyDismiss() {
         // Should test resign First responder
```

Comment:
Is there an issue related to this comment?

Reply:
no, and I don't know what this means, but I didn't change it either. @Daeda88 can maybe tell you.